js_modules/dagster-ui/packages/ui-core/src/overview/OverviewPageHeader.tsx

```diff
@@ -0,0 +1,42 @@
+import {Alert, Box, Heading, PageHeader} from '@dagster-io/ui-components';
+import React, {useContext} from 'react';
+
+import {OverviewTabs} from './OverviewTabs';
+import {CloudOSSContext} from '../app/CloudOSSContext';
+import {useAutoMaterializeSensorFlag} from '../assets/AutoMaterializeSensorFlag';
+import {useStateWithStorage} from '../hooks/useStateWithStorage';
+
+export const OverviewPageHeader = ({
+  tab,
+  queryData,
+  refreshState,
+  ...rest
+}: React.ComponentProps<typeof OverviewTabs> &
+  Omit<React.ComponentProps<typeof PageHeader>, 'title'>) => {
+  const {showGlobalAMPToSensorAMPBanner} = useContext(CloudOSSContext);
+  const [isBannerHidden, setIsBannerHidden] = useStateWithStorage<'1' | undefined>(
+    'banner:showGlobalAMPToSensorAMPBanner',
+    (key) => key,
+  );
+  const isSensorAMP = useAutoMaterializeSensorFlag() === 'has-sensor-amp';
+  return (
+    <PageHeader
+      title={<Heading>Overview</Heading>}
+      tabs={
+        <Box flex={{direction: 'column', gap: 8}}>
+          <OverviewTabs tab={tab} queryData={queryData} refreshState={refreshState} />
+          {isSensorAMP && showGlobalAMPToSensorAMPBanner && isBannerHidden !== '1' ? (
+            <Alert
+              title="Looking for the Auto-materialize tab?"
+              description="You can find your auto-materialize sensors on the Sensors tab."
+              onClose={() => {
+                setIsBannerHidden('1');
+              }}
+            />
+          ) : null}
+        </Box>
+      }
```

Comment:
This all looks good! In the spirit of the .cloud / .oss idea, I wonder if this should move to it's own file and have the PageHeader render a generic `<OverviewPageAlerts />`?

I imagine this will be fairly short lived so it may not be worth changing -- this implementation also seems fine :)

Reply:
Yep will rebase this PR to do that